src/store/actions/index.js

```diff
@@ -97,3 +101,21 @@ export const logout = (history) => (dispatch) => {
   localStorage.removeItem("user_id");
   history.push("/login");
 };
+
+export const GET_VIDID_START = "GET_VIDID_START";
+export const GET_VIDID_SUCCESS = "GET_VIDID_SUCCESS";
+export const GET_VIDID_ERROR = "GET_VIDID_ERROR";
+
+export const getVidID = (videoId) => (dispatch) => {
+  dispatch({ type: GET_VIDID_START });
+  axiosWithAuth()
+    .get(`/videos/${videoId}`)
+    .then((res) => {
+      dispatch({ type: GET_VIDID_SUCCESS, payload: res.data });
+      console.log("line110_+ACTIONS--GET_VIDID_SUCCESS", res.data);
+    })
+    .catch((err) => {
+      dispatch({ type: GET_VIDID_ERROR });
+      console.log(err);
+    });
+};
```

Comment:
We'll want to pull this into our useEffect in the actual singleVideoPage logic instead of duplicating what we have in here